### PR TITLE
Updated the readme file

### PR DESCRIPTION
**Pull request by Yanis42**
Added and edited links, updated instructions

Feel free if you have any suggestions, I will wait few days before merging this

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,28 @@
...
+**Discord:** <https://discord.gg/brETAakcXr>
+
+## Zelda Decompilation
 **Website:** <https://zelda64.dev>
 
 **Discord:** <https://discord.zelda64.dev>
 
 ## Installation
 
+Note: instructions for setting up Visual Studio Code can be found [here](docs/vscode.md).
```
Comment (thecozies) [inline README.md:24]: yooo nice

**Review by thecozies — APPROVED** [review]
looks great!

**Comment by Yanis42** [comment]
Merging this tomorrow unless someone has something to say

File: README.md (hunk 1 above)
Comment (gheskett) [reply README.md:24]: You think there's any merit to linking the HackerSM64 wiki tutorial for WSL2 installation or is the OoT process too different to make sense?

File: README.md [hunk 2]
```diff
@@ -1,44 +1,28 @@
-# The Legend of Zelda: Ocarina of Time
+# Hacker Zelda: Ocarina of Time (HackerOoT)
```
[the remark this answers is not in the document]
Comment (Yanis42) [reply README.md:1]: I'm writing ``OoT`` out of habit, I totally understand if that bothers people, I think it's fine for now though as other files would need to be updated

I'll ask people on Discord for their opinion, that's not really something I can decide alone imo

**Comment by Yanis42** [comment]
changed OOT to OoT everywhere, should be good to go now 👍

**Pull request merged**